Sources/MessagingInApp/Views/UIKitInline.swift

```diff
@@ -62,6 +62,10 @@ public class InlineMessageUIView: UIView, GistInlineMessageUIViewDelegate {
         // An element id will not be set yet. No need to check for messages to display.
     }
 
+    deinit {
+        inAppMessageView?.teardownView()
```

Comment:
dismiss will be called via `GistInlineMessageUIView.deinit` ?

Reply:
Yup, that's correct